core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -907,6 +926,18 @@ export class ConversationSkinComponent {
                     story_url_fragment: storyUrlFragment
                   });
             }
+            this.learnerDashboardBackendApiService
+              .fetchLearnerCompletedChaptersCountDataAsync().then(
+                (responseData) => {
+                  let newCompletedChaptersCount = (
+                    responseData.completedChaptersCount);
+                  if (
+                    newCompletedChaptersCount !== this.completedChaptersCount
```

Comment:
Wasn't `+ 1` here before?

Reply:
Yes, but I changed it because of the 80 character limit.

Additionally, as a side-effect, this now covers a highly unlikely, but possible case:

If a learner, who completed 3 chapters, begins a new chapter A, and in the middle of this, begins and completes a different chapter B in a different tab, and then comes back to A and completes it, they have now completed 5 chapters and should see the message.

But the completed chapters count when chapter A was initially loaded was 3, and because the new completed count is 5, the old condition ( of +1) would not have worked and the message wouldn't have been displayed.

The inequality condition doesn't suffer from that issue.